docker/entrypoint.sh

```diff
@@ -1,5 +1,14 @@
 #!/usr/bin/env bash
 
 set -e
+set -m # enable job control
 
-go run ./internal/nitro-network/main.go
+go run ./scripts/start-rpc-servers.go &
```

Comment:
Is there any way we could support running this container against an external chain, by accepting parameters somehow? In our boost integration we do this with [env vars](https://github.com/statechannels/boost/blob/2d532f0271f90888ff1ff6b997cad47de27b434f/docker/devnet/booster-http/entrypoint.sh#L14) but there may be a more "dockery" way of doing that.

That would let us use this dockerized nitro with a local filecoin devnet.

Reply:
That's a good point. Created https://github.com/statechannels/go-nitro/issues/1467.